content/en/docs/tasks/observability/metrics/lightstep/index.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Viewing Istio Metrics in Cloud Observability from ServiceNow
+description: This task shows you how to use OpenTelemetry to ingest and view Istio metrics in Cloud Observability (formerly Lightstep).
+weight: 11
+keywords: [telemetry, metrics ,lightstep, servicenow, cloud observability]
+aliases:
+ - /docs/tasks/observability/distributed-tracing/lightstep/
```

Comment:
I'm wondering if there is a format issue with number of leading spaces, as this does look correct. Need to try pulling your PR and trying the link-check locally.
```suggestion
    - /docs/tasks/observability/distributed-tracing/lightstep/
```

Reply:
Basically, the alias discussion is here: https://istio.io/latest/docs/releases/contribute/front-matter/#rename-move-or-delete-pages

Reply:
I did pull your branch locally and try a few things with the aliases, and I never got it to work. I guess I would just do option 1 and update the 1 link to point to the archive and remove the alias.

Reply:
@frankbu is there some magic on the `aliases:`?